Makefile

```diff
@@ -67,3 +67,20 @@ sonar/js:
 	unset SONARQUBE_SCANNER_PARAMS
 	sonar-scanner --debug
 
+.PHONY: go-bindata
+go-bindata:
+	@if which go-bindata > /dev/null; then \
+		echo "##### Updating go-bindata..."; \
+		cd $(mktemp -d) && GOSUMDB=off go get -u github.com/go-bindata/go-bindata/...; \
+	fi
```

Comment:
this way of updating bindata is not working locally

Reply:
For me, it is not working.. I have go version `go1.14.2`

Reply:
What error do you see?

Reply:
sometime it works..sometime not.. if i have go-bindata version 3.1.2.. it will not go and update go-bindata which it should do.. so we are always updating go-bindata in `go-bindata-check` endpoint-operator https://github.com/open-cluster-management/endpoint-operator/blob/master/Makefile#L91
so if there is diff, it will fail in pre-commit hook